karma.conf.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ *  Power BI Visualizations
+ *
+ *  Copyright (c) Microsoft Corporation
+ *  All rights reserved.
+ *  MIT License
+ *
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy
+ *  of this software and associated documentation files (the ""Software""), to deal
+ *  in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  copies of the Software, and to permit persons to whom the Software is
+ *  furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in
+ *  all copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *  THE SOFTWARE.
+ */
+
+"use strict";
+
+const webpackConfig = require("./test.webpack.config.js");
+const tsconfig = require("./test.tsconfig.json");
+const path = require("path");
+
+const testRecursivePath = "test/visualTest.ts";
+const srcOriginalRecursivePath = "src/**/*.ts";
+const srcRecursivePath = ".tmp/drop/**/*.js";
+const coverageFolder = "coverage";
+const globals = "./test/globals.ts";
```

Comment:
@MrMeison Could you include the globals.ts file into test env to init powerbi variable?
Example:
https://github.com/Microsoft/powerbi-visuals-mekkochart/commit/7ece44d1a694f4d24d93913ea62a6ce2694a016a#diff-20464016e9a426f5bd8cc5ac7f55e5f4R2

Reply:
Fixed